src/slim-select/render.test.ts

```diff
@@ -7,35 +7,1192 @@
 import { describe, expect, test } from '@jest/globals'
 import Render, { Callbacks } from './render'
 import Settings from './settings'
-import Store, { DataArray, Option } from './store'
+import Store, { Option } from './store'
 import CssClasses from './css_classes'
 
-describe('select module', () => {
-  test('constructor', () => {
-    // create a new store with 2 options
+describe('render module', () => {
+  let render: Render
+  let openMock: jest.Mock
```

Comment:
We really shouldnt need to mock anything. I really want to keep us actually testing things and for simple things like callback functions if we need to test whether it was triggered or not can be done without mocks.